src/main/kotlin/org/elm/workspace/ElmFormatUtils.kt

```diff
@@ -0,0 +1,69 @@
+package org.elm.workspace
+
+import com.intellij.execution.ExecutionException
+import com.intellij.execution.process.ProcessOutput
+import com.intellij.openapi.application.ApplicationManager
+import com.intellij.openapi.command.CommandProcessor
+import com.intellij.openapi.editor.Document
+import com.intellij.openapi.fileEditor.FileDocumentManager
+import com.intellij.openapi.progress.ProgressManager
+import com.intellij.openapi.project.Project
+import com.intellij.openapi.project.ProjectManager
+import com.intellij.openapi.roots.ProjectRootManager
+import com.intellij.openapi.vfs.VirtualFile
+import com.intellij.psi.PsiFile
+import org.elm.lang.core.psi.ElmFile
+import org.elm.openapiext.isSuccess
+import org.elm.openapiext.toPsiFile
+
+object ElmFormatUtils {
```

Comment:
All of the code in this file should be moved into `ElmFormatCLI`. Either as helper functions outside of the class or as methods on the companion object.